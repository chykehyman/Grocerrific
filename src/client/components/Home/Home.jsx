import React from 'react';
import { Link } from 'react-router-dom';


const HomePage = () => (
  <div className="jumbotron mt-4">
    <h1 className="display-4">STOP AND SHOP</h1>
    <p><i style={{ color: '#777' }}>Your one stop shop for your grocery items</i></p>
    <p className="lead">
      <Link to="/shop" className="btn btn-lg">Shop Now</Link>
    </p>
  </div>
);

export default HomePage;
