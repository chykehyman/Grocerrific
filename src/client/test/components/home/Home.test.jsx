import Home from '../../../components/Home/Home';


describe('<Home /> component test', () => {
  it('renders home page component without crashing', () => {
    const shallowWrapper = shallow(<Home />);
    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
