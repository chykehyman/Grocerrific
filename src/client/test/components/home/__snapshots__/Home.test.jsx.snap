// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> component test renders home page component without crashing 1`] = `
<div
  className="jumbotron mt-4"
>
  <h1
    className="display-4"
  >
    STOP AND SHOP
  </h1>
  <p>
    <i
      style={
        Object {
          "color": "#777",
        }
      }
    >
      Your one stop shop for your grocery items
    </i>
  </p>
  <p
    className="lead"
  >
    <Link
      className="btn btn-lg"
      replace={false}
      to="/shop"
    >
      Shop Now
    </Link>
  </p>
</div>
`;
